style/outsidein.css

```diff
@@ -112,38 +115,45 @@ body {
     width: 2.5rem;
 }
 
+.outside-in-container .outside-in-idi-Link,
+.outside-in-container .outside-in-Ontario-Government-Link {
+    color: #FFF;
+    font-size: 3rem;
+}
+
 .outside-in-container .outside-in-idi-Link:before {
     font-family: "outside-in-IDRC-text";
-    font-size: 3rem;
     content: "\e600";
-    color: #FFF;
 }
 
 .outside-in-container .outside-in-Ontario-Government-Link:before {
     font-family: "outside-in";
-    font-size: 3rem;
     content: "\e605";
-    color: #FFF;
 }
 
 .outside-in-container .outside-in-footerLogo a:before {
     font-family: "outside-in";
     content: "\e604";
-    color: #C0D660;
     padding-right: 0.5rem;
 }
 
 .fl-prefsEditor-buttons {
     font-size: 0.8rem;
     margin-left: 2rem;
+    margin-right: 4%;
 }
 
-.fl-prefsEditor-separatedPanel .fl-prefsEditor-showHide {
+.fl-prefsEditor-showHide {
     min-width: inherit;
 }
 
-.fl-prefsEditor-separatedPanel .fl-panelBar .fl-prefsEditor-buttons {
-    margin-right: 4%;
+.fl-prefsEditor-buttons button {
+    margin: 0.5rem;
+    padding: 0;
+}
+
+.fl-prefsEditor-buttons button:focus {
+    outline: 0.2rem solid #4c4c4c;
 }
 
 .outside-in-container .outside-in-TOC {
```

Comment:
The ToC font size is a bit bigger than the rest of the doc; can we make it smaller? Also, it's missing any hover styles, in the default theme and the high contrast themes (focus styling is ok).